src/Lykke.Service.ConfirmationCodes.AzureRepositories/Settings/AzureQueueSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.Service.ConfirmationCodes.AzureRepositories.Settings
+{
+    public class AzureQueueSettings
+    {
+        public string ConnectionString { get; set; }
```

Comment:
Add the check attribute

Reply:
https://github.com/LykkeCity/LykkeSettingsParser/blob/d84586f61d1c668a89879d3256b4682958b61ec5/src/Lykke.SettingsReader/SettingsProcessor/Attributes/AzureQueueCheckAttribute.cs